modules/k3s/main.tf

```diff
@@ -16,6 +16,7 @@ module "master" {
    server_flags=var.server_flags
    availability_zone=var.availability_zone
    sg_id=var.sg_id
+   volume_size=var.volume_size
```

Comment:
Suggestion to add a default value for volume size in case it's not provided.
Similar changes where applicable

Reply:
right now we dont have that even on rke2, this is a fix PR that i added this to be able to tests on minor.
we should open a task for those behaviors!

Reply:
Sure sound good to me